tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
What prompted this ?

Reply:
https://threejs.org/docs/index.html#manual/en/introduction/Typescript-setup 😃